src/ImageSharp/PixelFormats/PixelImplementations/Rgb24.cs

```diff
@@ -201,7 +201,17 @@ public void FromLa32(La32 source)
 
         /// <inheritdoc/>
         [MethodImpl(InliningOptions.ShortMethod)]
-        public void FromRgba32(Rgba32 source) => this = source.Rgb;
+        public void FromRgba32(Rgba32 source)
+        {
+#if NETSTANDARD2_0
+            // See https://github.com/SixLabors/ImageSharp/issues/1275
+            this.R = source.R;
+            this.G = source.G;
+            this.B = source.B;
+#else
+            this = source.Rgb;
+#endif
+        }
```

Comment:
Very nice! 🎉 

Remember someone talking about opening a dotnet/runtime issue on twitter, if it exists we should provide this info.

Reply:
I didn't open one. (I always struggle to provide a cut down solution to help them). Defo wise though.